Gruntfile.js

```diff
@@ -25,7 +25,9 @@ banner = "/*!\n" +
 umdStart = "(function( factory ) {\n" +
 	"\tif ( typeof define === \"function\" && define.amd ) {\n";
 
-umdMiddle = "\t} else {\n" +
+umdMiddle = "\t} else if (typeof exports === \"object\") {\n" +
```

Comment:
I think you want to add CommonJS pattern. If that so, you have to change this line to:
```js
} else if ( typeof module === 'object' && module.exports ) {
```